Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.6.3
+MAINTAINER Akky AKIMOTO <akimoto@gmail.com>
+
+# throw errors if Gemfile has been modified since Gemfile.lock
+RUN bundle config --global frozen 1
+
+WORKDIR /usr/src/app
+
+COPY Gemfile Gemfile.lock ./
+RUN gem update --system && \
+    gem install bundler && \
+    bundle update --bundler && \
+    bundle install
+COPY . .
+COPY for-docker.env .env
```

Comment:
you can have `for-docker.env` as `.env` in repo and remove this line
or better have environment passed in compose file and not have any .env file
```
environment:
  REDIS_URL:redis://redis:6379
```
I believe `REDIS_HOST` and `REDIS_PORT` are not in use